src/components/pages/Home/MainHeader.js

```diff
@@ -6,6 +6,25 @@ import UploadCase from '../Upload/UploadCase';
 import Icon from '@ant-design/icons';
 import Profile from '../../../styles/icons/profile.svg';
 import HRFLogo from '../../../styles/HRFlogo.png';
+import Notifications from './Notifications';
+
+const MOCK_NOTIFICATIONS = [
```

Comment:
Is this temporary?
I assume it is since it has an unconventional name. 
Perhaps add a comment here indicating its use if so.